Frameworks/CoreData/NSAttributeDescription.mm

```diff
@@ -14,8 +14,111 @@
 //
 //******************************************************************************
 
+#import <CoreData/CoreData.h>
+#import <Foundation/Foundation.h>
 #import <StubReturn.h>
-#import <CoreData/NSAttributeDescription.h>
+
+#import <CoreData/NSManagedObjectModel-XMLParsing.h>
+
+@interface NSAttributeDescription () {
+    StrongId<id> _minValue;
+    StrongId<id> _maxValue;
+}
+@end
```

Comment:
?
Our coding convention usually places ivars in the class extension block in the implementation file, which this is.
They're not exposed, though. min/max value are exposed only through the validation predicates.

---
In reply to: [70720550](https://github.com/Microsoft/WinObjC/pull/610#discussion_r70720550) [](ancestors = 70720550)